tests/component/sections/create-dataset/CreateDataset.spec.tsx

```diff
@@ -86,6 +93,36 @@ describe('Create Dataset', () => {
         cy.findByText('Not Implemented').should('exist')
       })
   })
+  it('pre-fills the form with user data', () => {
+    cy.customMount(
+      <SessionProvider repository={userRepository}>
+        <NotImplementedModalProvider>
+          <CreateDataset
+            repository={datasetRepository}
+            collectionId={'test-collectionId'}
+            metadataBlockInfoRepository={metadataBlockInfoRepository}
+          />
+        </NotImplementedModalProvider>
+      </SessionProvider>
+    )
+    cy.findAllByLabelText(/^Name/i).each(($el) => {
```

Comment:
What about finding Author sub fields or sub fields in general like [this](https://github.com/IQSS/dataverse-frontend/blob/develop/tests/component/sections/create-dataset/CreateDataset.spec.tsx#L22) 

Just thinking if it could be better but maybe not, what do you think?

Reply:
Yes, I like that better, it depends less on implementation, thanks :)